notebook-lm-clone/app.py

```
@@ -0,0 +1,970 @@
+    if 'pipeline_initialized' not in st.session_state:
+        st.session_state.pipeline_initialized = False
+
+def reset_chat():
+    try:
+        # Clear existing session from Zep if memory is available
+        # if st.session_state.pipeline and st.session_state.pipeline['memory']:
+        memory = st.session_state.pipeline['memory']
+        try:
+            memory.clear_session()
+            st.success("✅ Zep session cleared and recreated")
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)